pandas/_libs/window/indexers.pyx

```diff
@@ -15,7 +15,7 @@ def calculate_variable_window_bounds(
     int64_t window_size,
     object min_periods,  # unused but here to match get_window_bounds signature
     bint center,
-    object closed,
+    str closed,
```

Comment:
This could be None? does cython include implicit Optional?

Reply:
maybe update docstring. since we have 

```
    # default is 'right'
    if closed is None:
        closed = 'right'
```
in the code.

Reply:
> does cython include implicit Optional?

for cdef functions, implicit Optional is _always_ in effect